tech.json

```diff
@@ -895,6 +895,23 @@
             "Can be used to make it through an opening door, or barely just past a ledge."
           ]
         },
+        {
+          "name": "canXrayWaitForIFrames",
+          "requires": [ "XRayScope" ],
+          "note": [
+            "Use X-Ray to pause time until invulnerability frames are finished",
+            "Combine with Plasma Beam to repeatedly damage enemies with one fired shot, called Microwave",
```

Comment:
Didnt know this had any uses without charge, but looks like it does have some minimal ones